getting-started/vs-code-integration/snippets.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Telerik UI for Blazor Code Snippets
+page_title: Telerik UI for Blazor Code Snippets - Visual Studio Code Integration
+description: Learn how to add a new component  Templates.
+slug: getting-started-vs-code-integration-new-project
```

Comment:
Are we sure about this slug name? It doesn't appear to reflect the article content.

Reply:
Agreed, I accidentally deleted the first article and had to create a new one and forgot the slug, fixed